app/views/feedbacks/_main_fields.html.slim

```diff
@@ -1,9 +1,6 @@
-
 - [:goals, :achievements, :future, :comments, :conversation].each do |field|
   .row
-    .col-xs-12
-      = f.input field
+    .col-xs-12= f.input field
```

Comment:
:field ?

Reply:
ah no, not needed. nevermind

Reply:
This is part of a loop and `field` is a variable.